test/Standalone/transform/transform-convolutions.mlir

```diff
@@ -61,3 +61,90 @@ func.func @conv(%i: tensor<14x512x28x28xf32>, %f: tensor<1024x512x1x1xf32>,
                                 outs(%o: tensor<14x1024x28x28xf32>) -> tensor<14x1024x28x28xf32>
   return %0: tensor<14x1024x28x28xf32>
 }
+
+// -----
+
+transform.with_pdl_patterns {
+^bb0(%arg0: !pdl.operation):
+  sequence %arg0 failures(propagate) {
+    ^bb0(%arg1: !pdl.operation):
+      %0 = transform.structured.match ops{["linalg.conv_2d_nhwc_hwcf"]} in %arg1
+      %1 = transform.structured.generalize %0
+      // expected-error @below {{Could not map to matmul}}
```

Comment:
Interesting, I imagine that isn't `ConvLike` because it's not in `generic+matmul`, which is ok, but the error may be confusing. What is the error message here? Can we CHECK for it, just to make clear on the tests what they're testing?

Reply:
Here the problem is that you did not interchange, so you don't have an inner [parallel, parallel, reduction]. What would you like to check?

Reply:
It's not clear from each test what the problem is. Either a comment before the test explaining what's wrong with it, or some check (CHECK or expected-error) with a more clear error message.

If we don't have a clear error message (why it fails), then perhaps we should have?